Dockerfile

```diff
@@ -25,18 +30,25 @@ COPY --link Cargo.toml ./
 COPY --link Cargo.lock ./
 ARG CRATE=""
 ARG TARGETOS TARGETARCH TARGETVARIANT
+RUN curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- \
```

Comment:
We don't need to install WasmEdge manually anymore. That's take care automatically during the build process.

Reply:
done.